.github/workflows/template-build.yml

```
@@ -79,7 +82,7 @@ jobs:
         run: |
           # AppInsight - Connection String
           az config set extension.use_dynamic_install=yes_without_prompt
-          $AppInsightName = if ("${{ inputs.tag }}" -ieq "production") { "${{ env.APP_INSIGHT_PROD }}" } else { "${{ env.APP_INSIGHT_DEV }}" }
+          $AppInsightName = if ("${{ inputs.tag }}" -ieq "${{ env.PRODUCTION_TAG }}") { "${{ env.APP_INSIGHT_PROD }}" } else { "${{ env.APP_INSIGHT_DEV }}" }
```

```suggestion
          $AppInsightName = if ("${{ inputs.tag }}" -clike "pr-") { "${{ env.APP_INSIGHT_DEV }}" } else { "${{ env.APP_INSIGHT_PROD }}" }
```

It seems weird to use the production setting when the tag is staging.

Can we check for dev instead